tests/index.test.js

```diff
@@ -387,6 +387,13 @@ describe('startSauceConnect', () => {
         })
         expect(spawn.mock.calls).toMatchSnapshot()
     })
+
+    test('should contain custom headers', () => {
+        const SauceLabsNew = require('../src').default
+        const api = new SauceLabsNew({ headers: { 'user-agent': 'foo' } })
+        expect(util.inspect(api))
```

Comment:
instead of checking client object, let's make a request and see if `got` has received the correct header object.